cohere_aws/client.py

```diff
@@ -747,6 +747,72 @@ def create_finetune(
         bucket, old_short_key = parse_s3_url(s3_models_dir + job_name)
         s3_resource.Bucket(bucket).objects.filter(Prefix=old_short_key).delete()
 
+    def export_finetune(
+        self,
+        name: str,
+        s3_merged_weights_dir: str,
```

Comment:
```suggestion
        s3_checkpoint_dir: str,
```
For folks doing non-PEFT finetunes merged weights can be confusing

Reply:
Done. See [here](https://github.com/cohere-ai/cohere-aws/pull/168/commits/c946a2c798baf4fb42a3ecfc29f197ba62ee9586)